### PR TITLE
Save model to include metrics (fixed)

### PR DESCRIPTION
**Pull request by MikkelBaekgaardJoergensen**

### CODE REVIEW

**Review by askeko — CHANGES_REQUESTED** [review]
error functions should not be called "loss"

File: core/util/metrics.py [hunk 1]
```diff
@@ -1,26 +1,35 @@
 import math
+import torch
 
 
-def mae(y: list, y_hat: list) -> float:
+def mae(y: torch.Tensor, y_hat: torch.Tensor) -> float:
```
Comment (askeko) [inline core/util/metrics.py:5]: Should not be tensors

File: core/util/metrics.py [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
     for i in range(len(y)):
-        sum_tracker += abs(y[i] - y_hat[i])
-    return sum_tracker / len(y)
+        for j in range(len(y[i])):
+            sum_tracker += abs(y[i][j] - y_hat[i][j])
+    return float(sum_tracker / (len(y) * 24))
 
 
-def rmse(y: list, y_hat: list) -> float:
+def rmse(y: torch.Tensor, y_hat: list) -> float:
```
Comment (askeko) [inline core/util/metrics.py:15]: Should not be tensors

File: core/util/metrics.py [hunk 3]
```diff
@@ -1,26 +1,35 @@
...
     for i in range(len(y)):
-        sum_tracker += math.pow(y[i] - y_hat[i], 2)
-    return math.sqrt(sum_tracker / len(y))
+        for j in range(len(y[i])):
+            sum_tracker += math.pow(y[i][j] - y_hat[i][j], 2)
+    return math.sqrt(sum_tracker / (len(y) * 24))
 
 
-def smape(y: list, y_hat: list) -> float:
+def smape(y: torch.Tensor, y_hat: list) -> float:
```
Comment (askeko) [inline core/util/metrics.py:25]: Should not be tensors

**Review by askeko — CHANGES_REQUESTED** [review]

**Review by askeko — APPROVED** [review]
